tdrop

```diff
@@ -931,6 +931,8 @@ program_start() {
 			program_command+=(-e bash -c "$tmux_command")
 		elif [[ $program == kitty ]]; then
 			program_command+=(bash -c "$tmux_command")
+		elif [[ $program == wezterm || $class == wezterm ]]; then
+			program_command+=(start -- bash -c "$tmux_command")
```

Comment:
What is `start` here?  Does `bash -c "$tmux_command"` not work?

Reply:
`start` is wezterm's subcommand.
You can read more about it here: https://wezfurlong.org/wezterm/config/launch.html#launching-a-different-program-as-a-one-off-via-the-cli

Trying to execute bash directly on `wezterm` won't work (there will be an error informing you about incorrect usage). Well, at least on my flatpak version; I assume it's the same on others.
In theory wezterm supports tabs, panes and whatnot and tmux isn't needed, but some people still prefer to use it. It's the same with kitty. :p